_infra/helm/locust/locustfiles/locustfile.py

```diff
@@ -489,22 +508,26 @@ def on_start(self):
         self.sign_in()
 
     def sign_in(self):
-        response = self.get(url="/sign-in", expected_response_text="Sign in")
-        self.csrf_token = _capture_csrf_token(response.content.decode('utf8'))
-
-        response = self.post("/sign-in", data=_generate_random_respondent())
-        self.auth_cookie = response.cookies['authorization']
+        self.response = self.get(url="/sign-in", expected_response_text="Sign in")
+        self.csrf_token = _capture_csrf_token(self.response.content.decode('utf8'))
+        self.response = self.post(url="/sign-in",
+                                  data=_generate_random_respondent(),
+                                  allow_redirects=False,
+                                  expected_response_status=302)
+        self.auth_cookie = self.response.cookies['authorization']
 
     @task
     def perform_requests(self):
         for request in request_list:
-            grouping = None
+            grouping = request.get("grouping")
+            expected_response_text = request.get("expected_response_text")
+            expected_response_status = request.get("response_status", 200)
```

Comment:
Is this really `expected_response_status` (in the .json file?)

Reply:
Just being explicit